docs/_pages/contact/index.html

```diff
@@ -0,0 +1,35 @@
+---
+layout: contact
+title: contact
+subtitle: Get help via email, chat, and phone/video from the team that created Terratest.
+excerpt: Get help via email, chat, and phone/video from the team that created Terratest.
+permalink: /contact/
+slug: contact
+nav_title: Contact
+nav_title_link: /docs/
+custom_js:
+  - prism
+  - examples
+  - collection-browser_toc
+  - contact-form
+use_recaptcha: true
+---
+
+<div class="contact__page">
+  <div class="col-md-6 contact-column">
+    <h1>Contact Us</h1>
+    <p class="contact-subtitle">
+      Speak to a real human! 
+    </p>
+    <p class="contact-subtitle">
+      Use the form below or send an email to
+      <a href="mailto:info@terratest.io" >info@terratest.io.</a>
```

Comment:
```suggestion
      <a href="mailto:info@terratest.io" >info@gruntwork.io.</a>
```

Reply:
@brikis98 do you want to change mailto attribute also?

Reply:
Oh, yes, good catch!